src/BinaryParsers/PEBinary/PortableExecutable/PE.cs

```diff
@@ -802,6 +802,8 @@ public bool IsWixBinary
             }
         }
 
+        public bool IsARM64Binary => Machine == Machine.Arm64;

```

Comment:
>IsARM64Binary [](http://example.com/codeflow?start=20&length=13)

Let's remove this, as we don't have any other pattern which defines a helper for machine types. Instead, the rule can simply access the Machine property directly on this class.

Context here is that this PE abstraction has a bunch of legacy helpers on it that haven't been cleanly designed, and that I'd hoped would wither away. :)